### PR TITLE
Fix "Delete segment"

### PR DESCRIPTION
**Pull request by MontakOleg**
Delete segment option does not work because of errors on selection, if we ignore error, looks like all works fine:

Before

After

### CODE REVIEW

File: wme_street_to_river_plus.user.js [hunk 1]
```diff
@@ -482,7 +482,23 @@ function streetToRiver_init() {
...
+          // at v (third_party-45fe9aba9d649e1fe91a.js.gz:2:1169484)
+          // at y (third_party-45fe9aba9d649e1fe91a.js.gz:2:1169276)
+          // at d (third_party-45fe9aba9d649e1fe91a.js.gz:2:1167229)
+          // at u.trigger (third_party-45fe9aba9d649e1fe91a.js.gz:2:1169169)
+          // at a.trigger (app-2a31f5b5c73203749066.js.gz:1:800910)
+          // at a.triggerEvent (app-2a31f5b5c73203749066.js.gz:1:801095)
+          // at ct._triggerSelectionChanged (app-2a31f5b5c73203749066.js.gz:1:858504)
+          // at ct.selectFeatures (app-2a31f5b5c73203749066.js.gz:1:857818)
+          // at ct.setSelectedModels (app-2a31f5b5c73203749066.js.gz:1:858137)
+        console_log(err)
```
Comment (madnut-ua) [inline wme_street_to_river_plus.user.js:492]:
```suggestion
        console.log(err)
```

File: wme_street_to_river_plus.user.js [hunk 2]
```diff
@@ -482,7 +482,23 @@ function streetToRiver_init() {
...
       var riverLandmark_o = new wazeActionAddLandmark(riverLandmark);
       W.model.actionManager.add(riverLandmark_o);
-      W.selectionManager.setSelectedModels([riverLandmark]);
+      try {
+        W.selectionManager.setSelectedModels([riverLandmark]);
+      } catch (err) {
+        // Ignore error:
+          // Uncaught TypeError: Cannot read properties of undefined (reading 'children')
+          // at Object.WMETB_FPnewSelectionAvailable (FancyPermalink.min.js:216:184)
+          // at v (third_party-45fe9aba9d649e1fe91a.js.gz:2:1169484)
```
Comment (madnut-ua) [inline wme_street_to_river_plus.user.js]: I guess, 2 lines should be enough, the rest of callstack can be removed

File: wme_street_to_river_plus.user.js (hunk 1 above)
Comment (MontakOleg) [reply wme_street_to_river_plus.user.js:492]: `console_log` used in this script for logging, it defined in https://github.com/MontakOleg/wme-street-to-river-plus-mod/blob/fix-segment-deletion/wme_street_to_river_plus.user.js#L1028:L1033

File: wme_street_to_river_plus.user.js (hunk 2 above)
Comment (MontakOleg) [reply wme_street_to_river_plus.user.js]: Sounds reasonable, done

**Review by madnut-ua — APPROVED** [review]

**Pull request merged**